packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -117,6 +114,12 @@ class IrisGridTableModelTemplate<
 
   static COLUMN_BUFFER_PAGES = 0;
 
+  static isIrisGridTableModelTemplate(
```

Comment:
Instead of making it a static, export it as it's own function defined outside of the class.